react/Wrapper.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
   selectedSeller: Seller | undefined
   text?: string
   unavailableText?: string
+  onClickBehavior?: string
```

Comment:
Update the type to include only the two possible values, as requested in the `AddToCartButton.tsx` file 😃 .